### ISSUE TITLE
Button "hover" effect

### ISSUE THREAD

**Issue by jibin1573** [LMMS/lmms#4464]
Can we have glowing buttons, like in FL studio. Like when we press a button, it should light up with a different colour. 

Thank you!

**Comment by Sawuare** [comment]
In LMMS 1.2.0 RC6, all buttons glow when pressed.

**Comment by jibin1573** [comment]
I mean the buttons should light up. Please check this theme, and you'll understand. Link: https://lmms.io/lsp/?action=show&file=11356

(After downloading this file, you have to convert the file to tar.bz2)

**Comment by Sawuare** [comment]
AFAIK, the draw mode and edit mode buttons have 3 states: hover, pressed, checked. When a button is checked, it doesn't respond to hover/pressed. That's why a checked button doesn't glow when hovered/pressed.

I don't think we can change this behavior, but I think it makes sense.

**Comment by jibin1573** [comment]
@Sawuare, I wish if the default theme could do that. This same property is applied even in LMMS Classic theme.

**Comment by PhysSong** [comment]
It is possible with CSS:

https://user-images.githubusercontent.com/12591074/151691174-718bb896-8c64-4bf6-a4dc-dd9fbb121068.mp4

For test, I used this. Better colors should be chosen later, though.
```css
QToolButton:hover:checked {
	border-top: 1px solid #343840;
	border-bottom: 1px solid #4a515e;
	border-radius: 2px;
	background: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 #292d33, stop:1 #22262c)
}
```

**Comment by Rossmaxx** [comment]
just happened to come across this. this is a nice to have feature and phys' implementation looks good. one suggestion - the hover mode could get a lighter color (but not too light to hide the actual icon).

**Comment by luzpaz** [comment]
Is this as simple as adding the the above CSS code in to [style.css](lmms/blob/master/data/themes/default/style.css) ?

**Comment by DomClark** [comment]
It's a little more than that, but not much. It would need to be added to `style.css` for both the default and classic themes, and we'd want to discuss quite which colours to use.

**Comment by Rossmaxx** [comment]
I have opened #7202 which implements the hover effect, but I haven't decided on the colour scheme yet. Reviews appreciated.

**Comment by Rossmaxx** [comment]
Fixed in #7202
